tests/ext/peer_service_remapping.phpt

```diff
@@ -0,0 +1,60 @@
+--TEST--
+Properly remap peer service's sources based on DD_TRACE_PEER_SERVICE_MAPPING
+--ENV--
+DD_TRACE_PEER_SERVICE_MAPPING=foo:bar,db.instance:db,xyz:abc,db1:database,net.peer.name:net
```

Comment:
Looking at the code I think this case is well covered, but I would also add a test case where users set a wrong value, e.g. `DD_TRACE_PEER_SERVICE_MAPPING=foo=bar,only_tag` and verify that your code does not break.